base/applications/cmdutils/where/miniconutils.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * PROJECT:     ReactOS command-line utilities
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Providing partial <conutils.h> functionality in smaller program size
```

Comment:
If this is the purpose then it is not in the WHERE command that it should be placed. Either where the usual conutils.h file is, or not in ROS code (since the purpose is to actually use the library).